hippunfold/workflow/rules/download.smk

```diff
@@ -3,18 +3,43 @@
 download_dir = get_download_dir()
 
 
-rule download_extract_atlas:
-    params:
-        url=lambda wildcards: config["resource_urls"]["atlas"][wildcards.atlas],
-    output:
-        unzip_dir=directory(Path(download_dir) / "atlas" / "{atlas}"),
-    container:
-        config["singularity"]["autotop"]
-    shadow:
-        "minimal"
-    shell:
-        "wget 'https://{params.url}' -O temp.zip && "
-        " unzip -d {output.unzip_dir} temp.zip"
+# rule download_extract_atlas:
+#     params:
+#         url=config["resource_urls"]["atlas"][config["atlas"]],
+#     output:
+#         unzip_dir=Path(download_dir) / "atlas" / config["atlas"],
+#         label_gii=Path(download_dir)
```

Comment:
removing the named outputs (label_gii etc..) might cause issues with the dag.. I'll see what can be done..